### ISSUE TITLE
Powershell payloads not working with win7?

### ISSUE THREAD

**Issue by supergrapeman** [trustedsec/unicorn#89]
Generating Powershell payloads seems to work fine on win10, but not on win7. No reverse connection back to the listening meterpreter. Seems to be same behaviour on win7 32bit and win7 64 bit.

Any ideas?

(Thanks for the awesome work, by the way!)

**Comment by trustedsec** [comment]
Let me take a peek, also can you try instead of /w 1 /C change it to -w 1 and -C? Just want to rule out something. I need to build a win7 box to test :)

**Comment by supergrapeman** [comment]
Rats, I wasn't clear - sorry. 

I meant to write 'generated', not 'generating'. So I've generated the Powershell payloads under Kali, but when they run on win7, I get no connection (but it's fine with win10).

**Comment by trustedsec** [comment]
Can you try the /w 1 thing above by any chance? If it still doesn't call back let me know I'll take a look

**Comment by supergrapeman** [comment]
Thanks, I tried tweaking that, and replacing /w 1 /C with -w 1 -C in the powershell_attack.txt which was created, but the result was the same.

When I paste in the command prompt in Windows, it runs, but doesn't appear to do anything, and closes the command prompt. I ran a quick tcpdump to see if it's sending any packets to the box with the metasploit listener, but there was no traffic.

I don't think it's fun and games with Windows Defender, as I've disabled the realtime scan on that, just in case.
